### ISSUE TITLE
Error when training: "Found array with 0 sample(s) (shape=(0, 143)) while a minimum of 1 is required by the normalize function."

### ISSUE THREAD

**Issue by Glavin001** [Glavin001/IssueBot#11]
`issues.json`: https://gist.github.com/Glavin001/0e49caff5758ce76032e8e42388e98fd

```
$ python scripts/multi-label_classification.py
Traceback (most recent call last):
  File "scripts/multi-label_classification.py", line 27, in <module>
    results = classifier.train_issues(user, repo, issues, ignore_labels)
  File "scripts/../src/classifier/classifier.py", line 269, in train_issues
    predicted_test = predict_with_classifier(classifier, x_test, lb)
  File "scripts/../src/classifier/classifier.py", line 78, in predict_with_classifier
    predicted = classifier.predict(x)
  File "/usr/local/lib/python2.7/site-packages/sklearn/utils/metaestimators.py", line 37, in <lambda>
    out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/sklearn/pipeline.py", line 203, in predict
    Xt = transform.transform(Xt)
  File "/usr/local/lib/python2.7/site-packages/sklearn/feature_extraction/text.py", line 1040, in transform
    X = normalize(X, norm=self.norm, copy=False)
  File "/usr/local/lib/python2.7/site-packages/sklearn/preprocessing/data.py", line 1280, in normalize
    estimator='the normalize function', dtype=FLOAT_DTYPES)
  File "/usr/local/lib/python2.7/site-packages/sklearn/utils/validation.py", line 407, in check_array
    context))
ValueError: Found array with 0 sample(s) (shape=(0, 143)) while a minimum of 1 is required by the normalize function.
```

**Comment by devindersinghsaggu** [comment]
@Glavin001 I am facing the same error in model.fit(x_train,y_train)) function.

Could you help me to resolve the same ?

**Comment by Glavin001** [comment]
@devindersinghsaggu Looking at the commit which closed this issue: https://github.com/Glavin001/IssueBot/commit/70f0559059ca6d93e6b4b98bb62b9e1ea026b152

> Fixes #11. Fix case where there were no unlabelled issues to test

Any chance you have an empty array, like I did with `x_test` below?

```python
predicted_test = []
if len(x_test) > 0:
    predicted_test = predict_with_classifier(classifier, x_test, lb)
```

It has been years since I worked on this project, so I do not recall more specifics.
